src/main/java/com/joyent/manta/monitor/Application.java

```diff
@@ -114,8 +124,17 @@ public static void main(final String[] args) throws InterruptedException {
                 Class<MantaOperationsChain> chainClass =
                         (Class<MantaOperationsChain>)Class.forName(runner.getChainClassName());
                 MantaOperationsChain chain = injector.getInstance(chainClass);
+                Histogram requestPutHistogram = Histogram.build()
+                        .name("manta_monitor_put_request_latency_seconds_"
+                                + chain.getClass().getSimpleName())
+                        .help("Put Request latency in seconds")
+                        .create();
+                // Add the histogram here, to register it later below
+                requestPutHistogramList.add(requestPutHistogram);
```

Comment:
Why are you using a list here when you have a `Map` instance in which you can just call `.values()` on?